tools/core_packages.txt

```diff
@@ -1,74 +1,104 @@
-acl
-attr
-binutils
 brotli
 bz2
-c_ares
 ca_certificates
+c_ares
 crew_profile_base
-curl
+docbook_xml
+docbook_xml412
+docbook_xml42
+docbook_xml43
+docbook_xml44
+docbook_xml45
+docbook_xml50
+docbook_xml51
+docbook_xsl
+docbook_xsl_nons
 e2fsprogs
 elfutils
 expat
 filecmd
 flex
-gdbm
+gcc
 gcc11
+gdbm
 gettext
 git
 glibc
+gmp
 gnutls
 groff
 icu4c
+isl
+jansson
+jemalloc
 krb5
 less
 libarchive
+libcurl
 libcyrussasl
 libdb
 libedit
 libffi
+libgcrypt
+libgpgerror
 libidn2
 libmetalink
+libmicrohttpd
 libnghttp2
 libpipeline
 libpsl
 libseccomp
 libsigsegv
+libssh
 libssp
 libtasn1
 libtirpc
+libunbound
 libunistring
 libxml2
+libxslt
 libyaml
+llvm
```

Comment:
This is in core now?  Are you sure?  If it is, we need to remove it immediately.

Reply:
The list is generated by `crew deps core`, so it might be correct?

Reply:
We need to remove it then.  There should be no reason to include this in core.  Not only is it massive but it's only needed for builds so it really should be in buildessential.